tools/disasm/sym_info.py

```diff
@@ -0,0 +1,217 @@
+#!/usr/bin/env python3
+# SPDX-FileCopyrightText: 2024 Dragorn421
```

Comment:
why not zeldaret?

Reply:
I copypasted my template and didn't think of it, will change